django_tasks/backends/database/backend.py

```diff
@@ -88,3 +91,18 @@ async def aget_result(self, result_id: str) -> TaskResult:
             return (await DBTaskResult.objects.aget(id=result_id)).task_result
         except (DBTaskResult.DoesNotExist, ValidationError) as e:
             raise ResultDoesNotExist(result_id) from e
+
+    def check(self, **kwargs: Any) -> List[CheckMessage]:
+        if "django_tasks.backends.database" not in settings.INSTALLED_APPS:
```

Comment:
Suggestion:

```python
from django.apps import apps

if apps.is_installed("django_tasks.backends.database"):
```

This uses Django's actual APIs for checking if an app is installed

Reply:
That is way better. Thank you for pointing to that API